extensions/script/python/ExecutePythonProcessor.h

```diff
@@ -18,12 +18,16 @@
  * limitations under the License.
  */
 
-#ifndef NIFI_MINIFI_CPP_EXECUTEPYPROC_H
-#define NIFI_MINIFI_CPP_EXECUTEPYPROC_H
+#ifndef EXTENSIONS_SCRIPT_PYTHON_EXECUTEPYTHONPROCESSOR_H_
+#define EXTENSIONS_SCRIPT_PYTHON_EXECUTEPYTHONPROCESSOR_H_
 
-#include <concurrentqueue.h>
-#include <core/Resource.h>
-#include <core/Processor.h>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <concurrentqueue.h> // NOLINT
+#include <core/Resource.h> // NOLINT
```

Comment:
Linter detects these as C headers because  "<" is used instead quotes.

Reply:
I replaced the `<>` with "", but for concurrentqueue, one gets a different error:
> 28:1  error  cpplint  Include the directory when naming .h files  [build/include_subdir] [4]

Reply:
I would simply disable that error, makes no sense to me.

Reply:
Noted :)